src/ngraph/frontend/onnx_import/ops_bridge.cpp

```diff
@@ -154,6 +154,12 @@ namespace ngraph
             return result;
         }
 
+        OperatorSet OperatorsBridge::_get_operator_set(const std::string& domain)
+        {
+            std::int64_t version = 10;
```

Comment:
Wouldn't be better to just set the `version` to `OperatorsBridge::LATEST_SUPPORTED_OPSET_VERSION` in the function call below?

Reply:
Agree with Artur comments below. This function overload is redundant. Since this is pimpl just modify _private_ `_get*` function to accept `domain` as a first parameter and set `version` parameter default value equal to: `OperatorsBridge::LATEST_SUPPORTED_OPSET_VERSION`.